test/integration/documentation_controller_branding_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_plugin_helper'
+
+class DocumentationControllerBrandingTest < ActionDispatch::IntegrationTest
+  def test_docs_redirect_branded
+    get "/links/docs/Managing_Hosts?chapter=registering-a-host_managing-hosts"
+
+    assert_redirected_to %r{https://access\.redhat\.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts}
```

Comment:
Why is this a regex? Wouldn't this work?
```suggestion
    assert_redirected_to "https://access.redhat.com/documentation/en-us/red_hat_satellite/#{ForemanThemeSatellite.documentation_version}/html/managing_hosts/registering_hosts_to_server_managing-hosts#Registering_Hosts_managing-hosts"
```

Reply:
Because the place I copied the idea from had a regex
Will test with a string before merging